core/commands/bitswap.go

```diff
@@ -16,7 +16,7 @@ import (
 
 var BitswapCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline:          "A set of commands to manipulate the bitswap agent",
+		Tagline:          "A set of commands to manipulate the bitswap agent.",
```

Comment:
nvm, I raised the issue, so here it is https://github.com/ipfs/go-ipfs/pull/2231